examples/e2e-tests/kubernetes/coreos/coreos.json

```diff
@@ -0,0 +1,126 @@
+{
+  "apiVersion": "vlabs",

```

Comment:
are the files in `examples/coreos` still relevant?

Reply:
these examples are mentioned in https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md under "distro"

Reply:
yes, excepting that the default model will still deliver some busted pods (keyvault and blobfuse installers).

Reply:
You can fix that by adding an add'l `&&` condition for "is not coreos" to the wiring that determines whether or not to install addons by default. See where that happens for keyvault flexvol here:

https://github.com/Azure/aks-engine/blob/master/pkg/engine/artifacts.go#L74

Here's a usage example where a more interesting boolean evaluation is used to determine whether or not to install by default:

https://github.com/Azure/aks-engine/blob/master/pkg/engine/artifacts.go#L113

I'd go through all of those addons and add the same "don't install by default if CoreOS" implementation to all that we know won't work w/ CoreOS.

Reply:
Definitely planning to do this before merge, was the main reason I left as WIP. Slowed down a bit because i'm on call this week 😄 hoping to finish up that validation this afternoon.